src/main/fc/rc.c

```diff
@@ -587,12 +587,12 @@ FAST_CODE void processRcCommand(void)
             }
             rawSetpoint[axis] = constrainf(angleRate, -1.0f * currentControlRateProfile->rate_limit[axis], 1.0f * currentControlRateProfile->rate_limit[axis]);
             DEBUG_SET(DEBUG_ANGLERATE, axis, angleRate);
-        } 
 
-        // adjust un-filtered setpoint steps to camera angle (mixing Roll and Yaw)
-        if (rxConfig()->fpvCamAngleDegrees && IS_RC_MODE_ACTIVE(BOXFPVANGLEMIX) && !FLIGHT_MODE(HEADFREE_MODE)) {
-            scaleSetpointToFpvCamAngle();
-        }
+            // adjust raw setpoint steps to camera angle (mixing Roll and Yaw)
```

Comment:
LGTM. Only question: Before this it runs after the loop, now it's using rawSetPoint* it's in the loop.

Reply:
Good point, yes, it should be outside the loop.  Will fix asap.